android/src/app/organicmaps/maplayer/MapButtonsController.java

```diff
@@ -268,10 +268,14 @@ public float getBottomButtonsHeight()
 
   public void showMapButtons(boolean show)
   {
-    if (show)
+    if (show) {
       UiUtils.show(mFrame);
```

Comment:
Please read our style guide and see how the surrounding code is formatted.

```suggestion
    if (show)
    {
      UiUtils.show(mFrame);
```